package-overrides/github/tushariscoolster/angular-nicescroll@0.0.9.json

```diff
@@ -0,0 +1,14 @@
+{
+  "main": "angular-nicescroll",
+  "registry": "jspm",
+  "dependencies": {
+    "angular": "jspm:angular@^1.4.8",
+    "jquery": "jspm:jquery@^2.1.4",
```

Comment:
Since the `registry` is set to `jspm`, this can just be `"angular": "^1.4.8"` and `"jquery": "^2.1.4"`.